wasm-schema-sandbox/www/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This is a Node application that serves the webpage, right? I imagine this is only necessary during development and that gh-pages can host/serve all the assets later. Is that true?

Reply:
Yes, this is only required during development